book/zappifest/plugins-manifest-format.md

```diff
@@ -189,7 +189,7 @@ Field Key                   | Description
 **description**             | Describes the plugin.
 **type**                    | Plugin type out of predefined types.
 **min_zapp_sdk**            | minimum zapp version required for the field, when the plugin is a multi platform, please specify the the platform in the following format: `"min_zapp_sdk": { "ios": "1.0.0", "android": "1.0.0" }`
-**whitelisted_account_ids** | Optional. Array of Applicaster's account_ids as individual strings. To find the relevent account_ids go to `accounts.applicaster.com`. If empty, plugin will be available across all accounts.
+**whitelisted_account_ids** | Array of Applicaster's account_ids as individual strings. To find the relevent account_ids go to `accounts.applicaster.com`. Please note - this is only updated on new plugin creation. Any changed afterwards must be managed by admins in the Zapp portal.
```

Comment:
I think that the sentence should be: `Please note - this is only updated on new plugin creation. Any changes afterward must be managed by the admins in the Zapp portal.`

What do you think?

Reply:
you are right - it's a spelling mistake